src/common/utils/model.ts

```diff
@@ -123,6 +130,48 @@ export function fetchAllCountyProjections(snapshotUrl: string | null = null) {
   return cachedCountiesProjections[key];
 }
 
+export async function fetchAllCountyProjectionsByState() {
+  // Query counties for states individually as the entire counties.timeseries.json is too large
+  // to be parsed by node.
+  const allProjections = await Promise.all(
+    regions.states.map(
+      async (state: State) => await fetchCountyProjectionsForState(state),
+    ),
+  );
+  return allProjections.flatMap(obj => obj);
+}
+
+/** Returns an array of `Projections` instances for all counties in a state. */
+const cachedCountiesProjectionsForState: {
+  [key: string]: Promise<Projections[]>;
+} = {};
+export function fetchCountyProjectionsForState(
+  state: State,
+  snapshotUrl: string | null = null,
+) {
+  snapshotUrl = snapshotUrl || getSnapshotUrlOverride();
+  async function fetch() {
+    const all = await new Api(
+      snapshotUrl,
+    ).fetchCountySummariesWithTimeseriesForState(state);
+    return all
+      .filter(summaryWithTimeseries => {
```

Comment:
I think we can drop this filter because this function should never be called for states not in our regions instance (this filtering originally showed up when e.g. we started adding territories to the backend but the frontend didn't know about them).

Reply:
This is actually filtering counties not states - I'd lean towards keeping it just to make sure we don't include counties that  aren't matching 1:1 (or we can continue with the strict call).  but can remove if you feel strongly

Reply:
Oh, I see what you mean.  Yeah, I guess I don't feel super strongly but I also can't immediately think of a case where the backend would have extra counties for a state and we would want to silently ignore that, so I probably lean towards removing this code, which it looks like you did in the follow-up PR.  Thanks!